presto-product-tests/src/main/resources/sql-tests/testcases/window_functions/windowSpecificationAABcp.sql

```diff
@@ -0,0 +1,8 @@
+-- database: presto; groups: window;
```

Comment:
I would merge `sql` and `result` files together. It will limit convention tests files and it is easier to read when you have a query and results in one place.

Moreover, (optionally - up to you) I would go one step further and merge all your window tests into one file. Here is a syntax of such convention tests:
```
-- database: hive; groups: example_smoketest,group2
-- delimiter: |; ignoreOrder: false; types: INTEGER|VARCHAR|INTEGER|VARCHAR
--! name: query_1
SELECT * FROM nation WHERE id=0
--!
0|ALGERIA|0| haggle. carefully final deposits detect slyly agai|
--! name: query_2
-- groups: additional_group
SELECT * FROM nation WHERE id=1
1|USA|1| foo bar|
--!
```